polar_route/route_planner.py

```diff
@@ -28,8 +30,50 @@ def _flattenCases(id,mesh):
             neighbour_indx.append(int(neighbour))
     return neighbour_case, neighbour_indx
 
-
-
+def _initialise_dijkstra_graph(dijkstra_graph):
+    '''
+        Initialising dijkstra graph information into a standard form
+    '''
+
+    dijkstra_graph_dict = {}
+    for idx,cell in dijkstra_graph.iterrows():
+        dijkstra_graph_dict[cell.name] = {}
+        dijkstra_graph_dict[cell.name]['id'] = cell.name
+        for key in cell.keys():
+            entry = cell[key]
+            if type(entry) == list:
+                entry = np.array(entry)
+            dijkstra_graph_dict[cell.name][key] = entry
+    return  dijkstra_graph_dict
+
+
+def _initialise_dijkstra_route(dijkstra_graph,dijkstra_route):
+    '''
+        Initialising dijkstra route into a standard path form
+    '''
+
+    org_path_points = np.array(dijkstra_route['geometry']['coordinates'])
+    org_cellindices = np.array(dijkstra_route['properties']['CellIndices'])
+    org_cellcases= np.array(dijkstra_route['properties']['cases'])
+
+    # -- Generating a dataframe of the case information -- 
+    Points      = np.concatenate([org_path_points[0,:][None,:],org_path_points[1:-1:2],org_path_points[-1,:][None,:]])
+    cellIndices = np.concatenate([[org_cellindices[0]],[org_cellindices[0]],org_cellindices[1:-1:2],[org_cellindices[-1]],[org_cellindices[-1]]])
+    cellcases = np.concatenate([[org_cellcases[0]],[org_cellcases[0]],org_cellcases[1:-1:2],[org_cellcases[-1]],[org_cellcases[-1]]])
+
+    cellDijk    = [dijkstra_graph[ii] for ii in cellIndices]
+    cells  = cellDijk[1:-1]
+    cases  = cellcases[1:-1]
+    aps = []
+    for ii in range(len(cells)-1):
+        aps += [find_edge(cells[ii],cells[ii+1],cases[ii+1])]
+
+    # #-- Setting some backend information
+    aps = aps
```

Comment:
`aps = aps` ?

Reply:
This has been removed in the next commit